vpr/src/base/route_constraint.cpp

```diff
@@ -0,0 +1,43 @@
+#include "route_constraint.h"
+
+RouteConstraint::RouteConstraint() {
+    net_name_ = std::string("");
+    net_type_ = std::string("");
+    route_method_ = std::string("");
+    is_valid_ = false;
+}
+
+void RouteConstraint::set_net_name(std::string name) {
+    net_name_ = name;
+    return;
+}
+
+std::string RouteConstraint::get_net_name() const {
+    return net_name_;
+}
+
+void RouteConstraint::set_net_type(std::string type) {
+    net_type_ = type;
+    return;
+}
+
+std::string RouteConstraint::get_net_type() const {
+    return net_type_;
+}
+
+void RouteConstraint::set_route_model(std::string route_method) {
+    route_method_ = route_method;
+    return;
+}
+
+std::string RouteConstraint::get_route_model() const {
```

Comment:
Let's follow our API naming convention. For instance ``get_route_model`` -> ``route_model``. All the accessors do not need any prefix ``get_``. Mutators do need ``set_`` or ``add_``. 
See example at 

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/libs/librrgraph/src/base/rr_graph_view.h

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/libs/librrgraph/src/base/rr_graph_builder.h

Reply:
fixed.